spec/factories/whitehall_export/index_document_factory.rb

```diff
@@ -4,6 +4,8 @@
 
     sequence(:document_id)
 
+    content_id { SecureRandom.uuid }
```

Comment:
Based on the change in https://github.com/alphagov/whitehall/pull/5417/files#diff-fe39f00137564632291307d578535423R15, the `content_id` will need to be nested under `document_information` (as it is in Whitehall).